pkg/standalone/common.go

```diff
@@ -47,6 +50,56 @@ func DefaultComponentsDirPath() string {
 	return path_filepath.Join(defaultDaprDirPath(), defaultComponentsDirName)
 }
 
+func DefaultResourcesDirPath() string {
+	return path_filepath.Join(defaultDaprDirPath(), defaultResourcesDirName)
+}
+
+// when either `components-path` or `resources-path` flags are not present then preference is given to resources dir and then components dir.
+// TODO: Remove this function and use `DefaultResourcesDirPath` when `--components-path` flag is removed.
+func DefaultResourcesDirPrecedence() string {
+	defaultResourcesDirPath := DefaultResourcesDirPath()
+	if _, err := os.Stat(defaultResourcesDirPath); os.IsNotExist(err) {
+		return DefaultComponentsDirPath()
+	}
+	return defaultResourcesDirPath
+}
+
 func DefaultConfigFilePath() string {
 	return path_filepath.Join(defaultDaprDirPath(), defaultConfigFileName)
 }
+
+// It used to copy existing resources from components dir to resources dir.
+// TODO: Remove this function when `--components-path` flag is removed.
+func moveFilesFromComponentsToResourcesDir(componentsDirPath, resourcesDirPath string) error {
+	if _, err := os.Stat(componentsDirPath); err == nil {
```

Comment:
Invert this please.

```
if ...; err != nil {
    return err
}
// proceed with logic
```
The caller should ensure that the src exists before calling this, otherwise we should rename the method and update the doc accordingly.